packages/mongo-nav/src/org-nav/OrgNav.tsx

```diff
@@ -185,44 +237,26 @@ function NavLinks({
     });
   }
 
-  let items: Array<React.ReactNode>;
-
-  if (showMoreDropdownMenu) {
-    items = navLinks.map(({ href, navId, text, testid }) => (
-      <MenuItem size="large" key={navId} href={href} data-testid={testid}>
-        <div className={css(`font-size: 16px;`)}>{text}</div>
-      </MenuItem>
-    ));
-  } else {
-    items = navLinks.map(({ href, navId, text, testid }) => (
-      <OrgNavLink
-        href={href}
-        loading={loading}
-        isActive={activeNav === navId}
-        className={rightLinkMargin}
-        key={navId}
-        onClick={onElementClick(navId)}
-        data-testid={testid}
-      >
-        {text}
-      </OrgNavLink>
-    ));
-  }
+  const items = navLinks.map(({ href, navId, text, testid }) => (
+    <OrgNavLink
+      href={href}
+      loading={loading}
+      isActive={activeNav === navId}
+      className={rightLinkMargin}
+      key={navId}
+      onClick={onElementClick(navId)}
+      data-testid={testid}
+    >
+      {text}
+    </OrgNavLink>
+  ));
 
   if (onPremVersion) {
-    if (showMoreDropdownMenu) {
-      items.push(
-        <VersionNumber style={VersionNumberStyle.Menu}>
-          {onPremVersion}
-        </VersionNumber>,
-      );
-    } else {
-      items.unshift(
-        <VersionNumber style={VersionNumberStyle.Nav}>
-          {onPremVersion}
-        </VersionNumber>,
-      );
-    }
+    items.unshift(
```

Comment:
This seems more clever than it needs to be. Can we just write this conditional and element within the JSX instead to ensure the correct ordering?

Reply:
👍